apps/webClient/src/views/Home/index.tsx

```diff
@@ -1,61 +1,139 @@
-import Logo from "../../components/Logo";
 
 interface Props {}
 
-const backgroundImage = `${process.env.PUBLIC_BUCKET}/fd-home.jpg`;
+const logo = `${process.env.PUBLIC_BUCKET}/fd-logo.png`;
 
 const useStyles = makeStyles((theme: Theme) => ({
-  contentWrapper: {
-    paddingBottom: 0,
-    width: 176
-  },
-  top: {
-    backgroundImage: `url("${backgroundImage}")`,
-    height: "100%",
-    backgroundRepeat: "no-repeat",
-    backgroundPosition: "center",
-    backgroundColor: "black"
-  },
-  futuredays: {
+  logo: {
+    height: "100px",
+  },
+  logoContainer: {
+    display: "flex",
+    justifyContent: "center",
+  },
+  middleContainer: {
+    display: "flex",
+    justifyContent: "space-between",
+    marginTop: "80px",
+  },
+  lowerContainer: {
+    display: "flex",
+    justifyContent: "space-around",
+    marginTop: "80px",
+  },
+  heroContainer: {
+    maxWidth: "300px",
+  },
+  bodyText: {
+    color: theme.palette.secondary.main,
+    textAlign: "center",
+    fontStyle: "italic",
+  },
+  aboutUsButton: {
+    backgroundColor: "white",
+    borderRadius: 0,
+    width: "60px",
+    height: "200px",
+  },
+  ourWorkButton: {
+    backgroundColor: "transparent",
+    borderRadius: 0,
+    border: "solid 4px white",
+    width: "60px",
+    height: "200px",
+  },
+  aboutButtonText: {
+    writingMode: "vertical-rl",
+    textOrientation: "sideways",
+    fontWeight: 500,
+  },
+  ourWorkButtonText: {
+    // Need to swap facing direction
+    writingMode: "vertical-rl",
+    textOrientation: "sideways",
+    fontWeight: 500,
+  },
+  partnerCard: {
+    maxWidth: "300px",
+    textAlign: "center",
+  },
+  heroHeader: {
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "flex-end",
     color: "white",
-    lineHeight: 1
-  },
-  titleContent: {
-    alignSelf: "flex-end",
-    marginBottom: theme.spacing(2),
-    marginRight: theme.spacing(2),
-    [theme.breakpoints.only("xs")]: {
-      backgroundColor: "black",
-      borderRadius: theme.spacing(2),
-      marginRight: 0
-    }
-  }
+    marginBottom: "10px",
+  },
+  whiteTitle: {
+    color: "white",
+    fontWeight: 500,
+    marginBottom: "10px",
+  },
 }));
 
 const Home: React.FC<Props> = ({}) => {
   const classes = useStyles();
-  const contentColor = "white";
 
   return (
-    <Grid container direction="row-reverse" className={classes.top}>
-      <Grid item className={classes.titleContent} sm={3}>
-        <Box pb={2} className={classes.contentWrapper}>
-          <Logo fill={contentColor} />
-          <Typography
-            className={classes.futuredays}
-            align="center"
-            variant="h5"
-          >
-            FutureDays
-          </Typography>
+    <Grid container style={{ margin: "8rem 0" }}>
+      <Grid item md xs></Grid>
+      <Grid item md={6} xs={10} lg={7}>
+        <Box className={classes.logoContainer}>
+          <img src={logo} alt="Future Days" className={classes.logo} />
+        </Box>
```

Comment:
I think we can get rid of lines 86, 87.

For line 88, instead of `Box`, we can probably just use more Grid:
```
<Grid container item ...centeringProps>
  <Logo />
</Grid>
```

Check the docs for proper centering props: https://material-ui.com/api/grid/